src/D2L.CodeStyle.Analysis/MutabilityInspector.cs

```diff
@@ -128,14 +137,8 @@ ITypeSymbol type
 					continue;
 				}
 				if( member is IMethodSymbol ) {

```

Comment:
I wasn't able to remove this, because of the cycle detection changes (I'd have to pass the stack along to the helper method, which just seemed wrong). Can you think of a different way that I'm not seeing?